schemas/ohi-jmx.yml

```diff
@@ -48,6 +48,8 @@
         - groovy
         - hirsute
         - bullseye
+        - bookworm
+
```

Comment:
Extremely picky comment about the extra blank line that is added here, feel free to ignore 😄

Reply:
Solved! Thanks